uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/installer/PlayStore.java

```diff
@@ -187,4 +187,49 @@ public void installApp(@NonNull final String searchHint) {
         installAppFromMarketPage();
     }
 
+    public void updateApp(@NonNull final String searchHint) {
```

Comment:
Consider my other suggestion and put this as interface method also on the `IAppInstaller`

Reply:
addressed in f9ecbb25a67397feee33a63807075d46e1b84038